### ISSUE TITLE
Constant loss after 5 iterations

### ISSUE THREAD

**Issue by kulkarnikeerti** [captanlevi/Contour-Detection-Pytorch#21]
Hi @captanlevi 

Thanks again. I was successfully able to start the training. However, the loss value becomes constant after 5 iterations. Could you please suggest me some possible solutions that I can try?

Thanks in advance
